docs/pages/configuration/overview.md

```diff
@@ -262,6 +262,16 @@ The port on which the development server will run. Defaults to `3000`. This opti
 
 If the port is already in use, the next available port will be used.
 
+### `canonicalUrl`
+
+Sets the canonical URL for your documentation site. This is used for SEO purposes and helps search engines understand the preferred version of a page.
+
+```json
+{
+  "canonicalUrl": "https://docs.example.com"
```

Comment:
It would be good to give an example that shows that the true value is `canonicalUrl + basePath + location`